README.md

```diff
@@ -52,7 +52,7 @@ tokenizer = YubiTokenizer()
     * [YubiBERT Small Encoder8](./yubiai/nlp/yubiEmbeddings/)
     * [HuggingFace Supported YubiTokenizers](./yubiai/nlp/tokenizer/)
 * Dec 2022
-    * [Yubi Text Language Detection](./yubiai/nlp/language_detection/)
+    * [YuLan - Yubi's Text Language Detection](./yubiai/nlp/language_detection/)
```

Comment:
Can you make it `[YuLan V1 - Yubi's Text Language Detection]` in Dec 2022
And `[YuLan V2 - Yubi's Text Language Detection]` in Feb 2023